apps/zui/scripts/merge-mac-release-files.mjs

```diff
@@ -0,0 +1,226 @@
+// Based on https://github.com/electron-userland/electron-builder/issues/5592#issuecomment-2004803764
+import { Octokit } from "octokit";
```

Comment:
Since nowhere else in the directory uses semicolons at the end of statements I think we should get rid of them in this file.